File: app/src/main/java/com/bhanit/tapthegrey/utils/constants/TapTheGrey.java
package com.bhanit.tapthegrey.utils.constants;

public interface TapTheGrey {
    String PREFERENCE_CITY_KEY = "city";
    String PREFERENCE_LOCATION_KEY = "location";

    interface NotificationConstants {
        String BUILDER = "builder";
    }

    interface Color {
        String GREY = "grey";
    }

    interface Time {
        int FIVE_SECOND = 5000;
        int ONE_SECOND = 1000;
        int NINE_HUNDERED_MILLI_SECOND = 900;
        int EIGHT_HUNDERED_MILLI_SECOND = 800;
        int SEVEN_HUNDERED_MILLI_SECOND = 700;
        int SIX_HUNDERED_MILLI_SECOND = 600;
        int FIVE_HUNDERED_MILLI_SECOND = 500;
        int FOUR_HUNDERED_MILLI_SECOND = 400;
        int THREE_HUNDERED_MILLI_SECOND = 300;
        int TWO_HUNDERED_MILLI_SECOND = 200;
        int ONE_HUNDERED_MILLI_SECOND = 100;

    }

    interface SharedPreferences {
        String CITY_LIST = "city_list";
        String LOCALITY_LIST = "locality_list";
        String IS_NEARBY_SELECTED = "is_nearby_selected";
        String LATITUDE = "latitude";
        String LONGITUDE = "longitude";
        String USER_SAVE = "patient_save";

    }
}

File: app/src/main/java/com/bhanit/games/tapthegrey/utils/constants/TapTheGrey.kt
package com.bhanit.games.tapthegrey.utils.constants

interface TapTheGrey {
    interface NotificationConstants {
        companion object {
            const val BUILDER: String = "builder"
        }
    }

    interface Color {
        companion object {
            const val GREY: String = "grey"
        }
    }

    interface LevelChange {
        companion object {
            const val LEVEL_CHANGE_SCORE_FOR_LEVEL_ONE: Int = 50
            const val LEVEL_CHANGE_SCORE_FOR_LEVEL_TWO: Int = 50
            const val LEVEL_CHANGE_SCORE_FOR_LEVEL_THREE: Int = 50
            const val LEVEL_CHANGE_SCORE_FOR_LEVEL_FOUR: Int = 50

            const val LEVEL_CHANGE_SCORE: Int = 80
        }
    }

    interface Time {
        companion object {
            const val FIVE_SECOND: Int = 5000
            const val ONE_SECOND: Int = 1000
            const val TWO_SECOND: Int = 2000
            const val NINE_HUNDERED_MILLI_SECOND: Int = 900
            const val EIGHT_HUNDERED_MILLI_SECOND: Int = 800
            const val SEVEN_HUNDERED_MILLI_SECOND: Int = 700
            const val SIX_HUNDERED_MILLI_SECOND: Int = 600
            const val FIVE_HUNDERED_MILLI_SECOND: Int = 500
            const val FOUR_HUNDERED_MILLI_SECOND: Int = 400
            const val THREE_HUNDERED_MILLI_SECOND: Int = 300
            const val TWO_HUNDERED_MILLI_SECOND: Int = 200
            const val ONE_HUNDERED_MILLI_SECOND: Int = 100
        }
    }

    interface Level {
        companion object {
            const val ONE: String = "one"
            const val TWO: String = "two"
            const val THREE: String = "three"
            const val FOUR: String = "four"
        }
    }

    interface LevelInInteger {
        companion object {
            const val ONE: Int = 1
            const val TWO: Int = 2
            const val THREE: Int = 3
            const val FOUR: Int = 4
        }
    }


    interface Count {
        companion object {
            const val SEVEN: Int = 7
            const val TEN: Int = 10
            const val TWENTY: Int = 20
            const val THIRTY: Int = 30
            const val FORTY: Int = 40
            const val FIFTY: Int = 50
            const val SIXTY: Int = 60
            const val SEVENTY: Int = 70
            const val EIGHTY: Int = 80
            const val NNETY: Int = 90
            const val HUNDRED: Int = 100
        }
    }

    companion object {
        const val WEBSITE: String = "https://bhanitgaurav.com/tapthegrey.html"
    }
}
